osfoffline/views/start_screen.py

```diff
@@ -1,12 +1,22 @@
+import furl
+import aiohttp
```

Comment:
Per PEP8, move installed packages below standard library modules. (furl and aiohttp below asyncio etc)

Within each category of imports, generally try to keep the imports alphabetical, to make it easier to find if/where something is imported.